awesomeConfig.cmake

```diff
@@ -137,11 +137,20 @@ pkg_check_modules(AWESOME_REQUIRED REQUIRED
     libstartup-notification-1.0>=0.10
     xproto>=7.0.15
     libxdg-basedir>=1.0.0
-    xcb-xrm)
+    xcb-xrm
+)
 
-if(NOT AWESOME_REQUIRED_FOUND OR NOT AWESOME_COMMON_REQUIRED_FOUND)
-    message(FATAL_ERROR)
-endif()
+# Check the deps one by one
+foreach(dependency ${AWESOME_DEPENDENCIES})
+    pkg_check_modules(TMP_DEP REQUIRED ${dependency})
+
+    if(NOT TMP_DEP_FOUND)
+        message(FATAL_ERROR)
+    endif()
+endforeach()
+
+# Do it again, but this time with the CFLAGS/LDFLAGS
+pkg_check_modules(AWESOME_REQUIRED REQUIRED ${AWESOME_DEPENDENCIES})
```

Comment:
Nice, thanks!
I assume that it lists all missing deps?
And in the end you would get an error for all of them when one fails with CFLAGS/LDFLAGS?